script/vendor-mimes

```diff
@@ -23,13 +48,27 @@ data.each do |mime, meta|
   next if extensions.empty?
   mimes[mime] = [] if mimes[mime].nil?
   mimes[mime].concat extensions
+
+  # Extract mime-types with "charset" metadata
+  charset_data[mime] = meta["charset"] if meta.key?("charset")
```

Comment:
We went from (current behavior) every MIME type gets the default charset to (proposed behavior) only a dozen or so get the default charset. I think we're missing some here. How would you feel about adding all `text/*` MIME types here?
```suggestion
  charset_data[mime] = meta["charset"] if meta.key?("charset")
  charset_data[mime] ||= "default" if mime.start_with?("text/")
```

Reply:
👍